cosmotheka/tests/test_mapper_quaia.py

```diff
@@ -0,0 +1,97 @@
+import numpy as np
+import cosmotheka as xc
+import healpy as hp
+import os
+
+
+def get_config():
+    return {'data_catalog': 'cosmotheka/tests/data/catalog.fits',
+            'selection_G': 'cosmotheka/tests/data/map.fits',
+            'selection_C': 'cosmotheka/tests/data/map.fits',
+            'mask_extra_G': 'cosmotheka/tests/data/map.fits',
+            'mask_extra_C': 'cosmotheka/tests/data/map.fits',
+            'z_name': 'Z',
+            'z_edges': [-1E-10, 1.0],
+            'path_rerun': '.',
+            'coords': 'C',
+            'nside': 32, 'mask_name': 'mask'}
+
+
+def cleanup_rerun():
+    for fname in ['Quaia_z-0.000_1.000_cat.fits',
+                  'Quaia_z-0.000_1.000_dndz.npz',
+                  'mask_mask_coordC_ns32.fits.gz',
+                  'Quaia_signal_map_coordC_ns32.fits.gz']:
+        if os.path.isfile(fname):
+            os.remove(fname)
+
+
+def get_mapper():
+    return xc.mappers.MapperQuaia(get_config())
+
+
+def test_smoke():
+    cleanup_rerun()
+    m = get_mapper()
+    m.get_catalog()
+    assert len(m.cat_data) == hp.nside2npix(32)
+    cleanup_rerun()
```

Comment:
More elegant solution than adding this at the end of each test is adding this at the beginning of the file

```
@pytest.fixture(autouse=True)
def teardown_test():
    clean_outdir()

def clean_outdir():
    os.system(f"rm -f {OUTDIR}*")
```
or change `clean_outdir` for `cleanup_rerun`

Reply:
Yep